app/css/views/themes.css

```diff
@@ -0,0 +1,43 @@
+.themes__new-link {
+  padding-top: .5rem;
+  height: 3rem;
+}
+
+.themes-list {
```

Comment:
Honestly all this top/bottom/line-height/absolute alignment look a bit messy, why not to use only left-right margins for nested elements if needed + ```align-items: center``` + top/bottom padding (if needed) on the ```li``` and ```flex: 1``` for ```themes-list__name```. Or I'm missing something?